lib/logstash/inputs/jdbc.rb

```diff
@@ -220,6 +250,10 @@ def execute_query(queue)
     # update default parameters
     @parameters['sql_last_value'] = @sql_last_value
     execute_statement(@statement, @parameters) do |row|
+      if enable_encoding?
```

Comment:
yeah. we could skip the method call and just do `if @enable_encoding` here... but fine if you prefer it like this.